uwblueprint/starter-code-v2
# Add Simple Entity Misc Updates (Swagger, E2E)

## Notion ticket link

N/A



## Implementation description
* Created copies of existing swagger entity endpoints for simple entity
* Created copies of existing entity E2E tests for simple entity



## Steps to test
1. Use the scrubber locally (yarn dev -t in starter-code-cli) with any backend, API, or database.2. 
2. Navigate to http://localhost:5000/api-docs/ 
<img width="1270" alt="image" src="https://user-images.githubusercontent.com/37782734/161392794-944888c7-b7a6-47ba-bc8a-6a020e00add2.png">

3. Verify that the endpoints on swagger for simple entity work

4. Run `pytest` in the e2e-tests folder, with command line arguments matching the stack chosen. Verify that they pass.


## What should reviewers focus on?
* 


## Checklist
